README.md

```diff
@@ -1,4 +1,4 @@
-Adafruit_MCP3008 [![Build Status](https://travis-ci.com/adafruit/Adafruit_MCP3008.svg?branch=master)](https://travis-ci.com/adafruit/Adafruit_MCP3008)
+Adafruit_MCP3008 [![Build Status](https://github.com/adafruit/Adafruit_MCP3008/workflows/Arduino%20Library%20CI/badge.svg)](https://github.com/adafruit/Adafruit_MCP3008/actions)
 ================
```

Comment:
@dherrada please update to use the updated layout seen here:
https://github.com/adafruit/Adafruit_ZeroPDM/tree/actionci